47159.4 <c-52#262> Created logger
47159.4 <c-52#262> appl_initialize was called
47159.4 <c-52#262:9> Logger was updated
47159.4 <c-52#262:9> Started intializing cube lib
47159.4 <c-52#262:9> Finished intializing cube lib with worker, starting to work
47159.5 <c-52#262:9> requestCubes => [49] Buffer size: 0
47159.5 <c-52#262:9> Sent requestCubes signal to root
47159.5 <c-52#262:9> Received 4 cubes from root
47159.5 <c-52#262:9> The main loop continues
47159.5 <c-52#262:9> Started solving a cube
47159.5 <c-52#262:9> Cube failed
47159.5 <c-52#262:9> Added failed cube
47159.5 <c-52#262:9> Started solving a cube
47159.5 <c-52#262:9> Skipped cube
47159.5 <c-52#262:9> Started solving a cube
47176.4 <c-52#262:9> Cube failed
47176.4 <c-52#262:9> Added failed cube
47176.4 <c-52#262:9> Started solving a cube
47714.8 <c-52#262:9> appl_interrupt was called
47714.8 <c-52#262:9> Interrupting worker
47714.8 <c-52#262:9> appl_interrupt was called
47714.8 <c-52#262:9> appl_withdraw was called
47714.8 <c-52#262:9> Started withdraw thread
47714.8 <c-52#262:9> STOP (0.00s since last cb)
47714.8 <c-52#262:9> Solving interrupted
47714.8 <c-52#262:9> The main loop continues
47714.8 <c-52#262:9> Leaving the main loop due to an interruption
47714.8 <c-52#262:9> Finished withdrawing lib
47714.8 <c-52#262:9> Enter destructor of CubeWorker
47714.8 <c-52#262:9> Time waiting for messages: 0.000
47714.8 <c-52#262:9> Enter destructor of CubeWorkerInterface
47714.8 <c-52#262:9> Enter destructor of CubeRoot
47714.8 <c-52#262:9> Finished withdraw thread
47775.8 <c-52#262:9> Enter destructor
47775.8 <c-52#262:9> Time working: 555.355
47775.8 <c-52#262:9> Time suspended: 0.000
47775.8 <c-52#262:9> Joined cleanup thread
47775.8 <c-52#262:9> Exit destructor
47775.8 <c-52#262:9> Destructing logger
